# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../procps/no_uptime.patch
# Copyright (C) 2009 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

diff -Nur procps-3.2.8-orig/Makefile procps-3.2.8/Makefile
--- procps-3.2.8-orig/Makefile	2009-08-08 13:42:13.000000000 +0100
+++ procps-3.2.8/Makefile	2009-08-08 13:42:56.000000000 +0100
@@ -45,13 +45,13 @@
 
 #SKIP     := $(bin)kill $(man1)kill.1
 
-BINFILES := $(usr/bin)uptime $(usr/bin)tload $(usr/bin)free $(usr/bin)w \
+BINFILES := $(usr/bin)tload $(usr/bin)free $(usr/bin)w \
             $(usr/bin)top $(usr/bin)vmstat $(usr/bin)watch $(usr/bin)skill \
             $(usr/bin)snice $(bin)kill $(sbin)sysctl $(usr/bin)pmap \
             $(usr/proc/bin)pgrep $(usr/proc/bin)pkill $(usr/bin)slabtop \
             $(usr/proc/bin)pwdx
 
-MANFILES := $(man1)uptime.1 $(man1)tload.1 $(man1)free.1 $(man1)w.1 \
+MANFILES := $(man1)tload.1 $(man1)free.1 $(man1)w.1 \
             $(man1)top.1 $(man1)watch.1 $(man1)skill.1 $(man1)kill.1 \
             $(man1)snice.1 $(man1)pgrep.1 $(man1)pkill.1 $(man1)pmap.1 \
             $(man5)sysctl.conf.5 $(man8)vmstat.8 $(man8)sysctl.8 \
